examples/step-57/doc/results.dox

```diff
@@ -0,0 +1,429 @@
+<h1>Results</h1>
+
+Now we use the method we discussed above to solve Navier Stokes equations with
+viscosity 1/400 and 1/7500.
+
+<h3> Test case 1: Re=400 </h3>
+
+In the first test case the viscosity is set to be 1/400. As we discussed in
+introduction, the initial guess is the solution to corresponding Stokes
+problem. In the following table, the residuals at each Newton's iteration on
+every mesh is shown. From the data in the table, the convergence rate
+of Newton's iteration is quadratic.
```

Comment:
Rephrase the last sentence: "The data in the table shows that Newton's iteration converges quadratically."